File: .net & mono/Weed3Demo/demo_plus/table/UserM.cs
using Noear.Weed;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Weed3Demo.test.table {
    public class UserM : DbTable {
        public UserM() : base(DbConfig.test) {
            table("users u");
            set("UserID", () => UserID);
            set("Nickname", () => Nickname);
            set("Sex", () => Sex);
            set("Icon", () => Icon);
            set("City", () => City);
        }

        public long? UserID;
        public String Nickname;
        public int? Sex;
        public String Icon;
        public String City;
    }
}

File: java/Weed3Demo/src/main/java/weed3demo/demo_plus/table/UserM.java
package weed3demo.demo_plus.table;

import org.noear.weed.DbTable;
import weed3demo.config.DbConfig;

/**
 * Created by noear on 2017/7/22.
 */
public class UserM extends DbTable {
    public UserM() {
        super(DbConfig.test);

        table("users u");
        set("UserID", () -> UserID);
        set("Nickname", () -> Nickname);
        set("Sex", () -> Sex);
        set("Icon", () -> Icon);
        set("City", () -> City);
    }

    public Long UserID;
    public String Nickname;
    public Integer Sex;
    public String Icon;
    public String City;
}
